### ISSUE TITLE
What definition of `float` does DDLm follow (CIF1.1 vs mmCIF)?

### ISSUE THREAD

**Issue by vaitkus** [COMCIFS/cif_core#384]
One of the goals of DDLm is to serve as a common format for small-molecule and protein crystallography. This is reflected by the fact, that DDLm versions of the dictionary contains data names from both worlds, often as aliases of the same concept (i.e. `_atom_site.B_iso_or_equiv`). However, I recently noticed that the `float` format defined by the mmCIF dictionary [1] and the one specified by the CIF1.1 grammar [2] are not compatible.

Besides some small differences (e.g. CIF1.1 allowing a leading '+' sign), the main difference is the order in which the exponent and the standard uncertainty parts appear, that is:
* CIF 1.1: `<numb><exp><su>`
* mmCIF: `<numb><su><exp>`

Up to now, I operated under the compliance to CIF1.1 when dealing with CIF files. Are there any guidelines on which format should be selected?

Alternatively, maybe it would be worthwhile to contact the mmCIF dictionary maintainers and ask them to change the float format so it more resembles the CIF1.1 one? The change might be feasible given that mmCIF files normally record standard uncertainties using separate data items rather than the parenthesis notation.

[1] https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Data/index.html, `float`
[2] https://www.iucr.org/resources/cif/spec/version1.1/cifsyntax, `<Float>`

**Comment by jamesrhester** [comment]
Yes, fortunately mmCIF standard style is to use a separate `su` data name everywhere, so this problem is hopefully easy to solve. We don't currently have a wwPDB representative on COMCIFS so I will need to sort that out first. Meanwhile, the CIF1.1 format should be preferred simply because that is the form you are overwhelmingly likely to encounter.
